Editor/Deployment/DeploymentSettings.cs

```diff
@@ -384,7 +384,12 @@ public async Task StartDeployment(ConfirmChangesDelegate confirmChanges)
             IReadOnlyDictionary<string, string> parameters = currentDeployer.HasGameServer
                 ? PrepareParameters(exeFilePath)
                 : PrepareGameParameter();
-            _parametersUpdater.Update(parametersPath, parameters);
+            var parameterUpdateResponse = _parametersUpdater.Update(parametersPath, parameters);
+            if (!parameterUpdateResponse.Success)
+            {
+                _logger.LogResponseError(parameterUpdateResponse);
```

Comment:
I see the other error scenarios in this method set a status box with appropriate error messaging. Is that relevant/needed here?

Reply:
Added

Reply:
Changed to use response message